feature_appview/src/main/java/cm/aptoide/pt/feature_appview/presentation/RelatedContentView.kt

```diff
@@ -0,0 +1,76 @@
+package cm.aptoide.pt.feature_appview.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.feature_appview.domain.model.RelatedCard
+import coil.compose.rememberImagePainter
+import coil.transform.RoundedCornersTransformation
+
+@Composable
+fun RelatedContentView(relatedContentList: List<RelatedCard>, listScope: LazyListScope?) {
```

Comment:
is this the same component as the editorial one? if yes we should change this name. Related content is in the context of app view, but these are Editorials.